src/Uno.UI.RuntimeTests/Tests/Uno_UI_Xaml_Core/Given_InputManager.cs

```diff
@@ -29,12 +29,6 @@ public class Given_InputManager
 #endif
 	public async Task When_VisibilityChangesWhileDispatching_Then_RecomputeOriginalSource()
 	{
-		if (Private.Infrastructure.TestServices.WindowHelper.IsXamlIsland)
```

Comment:
There are two methods with this `if` statement in this file

Reply:
Will remove them! :)